standards/technical-approach.md

```diff
@@ -43,8 +78,24 @@ Step 5 above includes the following steps to be followed by the Tech Lead/Archit
 2. If revisions are requested/necessary the document's author should make the necessary changes and record the process in a changelog to be added to the end of the document
 3. The above steps should be repeated until the document is approved
 
+### Changes After Approval
```

Comment:
This is great. I've also been thinking of this as an "Addendum" where we write updates to the TAD based on new findings.